test_added.go

```diff
@@ -0,0 +1,3 @@
+package reviewdog
+
+var TestAdded = 14
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported var TestAdded should have comment or be unexported